### ISSUE TITLE
Feature request Accurate calibration of goggle battery

### ISSUE THREAD

**Issue by fpvwannabee** [hd-zero/hdzero-goggle#161]
Bit of feedback based on my use of the hdzero goggles.
I've noticed that the goggle voltage is incorrectly reporting what my goggle battery is.
At the end of this morning's session they were stating 6.2 v on the osd but the battery when tested separately was 7v

**Comment by robhaswell** [comment]
Is this a high-capacity li-ion battery? The goggles draw a fair amount of power, definitely enough to sag some batteries by 0.4V per cell. You should measure the voltage of the battery while it's powering the goggles.

**Comment by robhaswell** [comment]
I've also noticed that my voltage needs calibrating.

**Comment by pitts-mo** [comment]
Agreed, we need an easy way to calibrate the current voltage reading.

However, These goggles consume a lot of power and I think you might be surprised to see how much voltage drop is occurring during goggle operation when you test voltage at the inside of the goggle connector VS directly at the battery... The cable length and the integrity of the connectors are MUCH more likely to influence voltage drop on HDZero goggle than most other goggles.

**Comment by Master92** [comment]
As #332 has been merged, I suppose you can now close this issue 🙂

**Comment by Master92** [comment]
@ligenxxxx could you be so nice and close this issue?
